packages/docusaurus-plugin-content-docs/src/lastUpdate.ts

```diff
@@ -5,69 +5,29 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import shell from 'shelljs';
 import logger from '@docusaurus/logger';
-import path from 'path';
+import {getCommitterDateForFile} from '@docusaurus/utils';
 
 type FileLastUpdateData = {timestamp?: number; author?: string};
 
-const GIT_COMMIT_TIMESTAMP_AUTHOR_REGEX = /^(?<timestamp>\d+),(?<author>.+)$/;
-
-let showedGitRequirementError = false;
-
 export async function getFileLastUpdate(
   filePath?: string,
 ): Promise<FileLastUpdateData | null> {
   if (!filePath) {
     return null;
   }
-  function getTimestampAndAuthor(str: string): FileLastUpdateData | null {
-    if (!str) {
-      return null;
-    }
-
-    const temp = str.match(GIT_COMMIT_TIMESTAMP_AUTHOR_REGEX)?.groups;
-    return temp
-      ? {timestamp: Number(temp.timestamp), author: temp.author}
-      : null;
-  }
 
   // Wrap in try/catch in case the shell commands fail
   // (e.g. project doesn't use Git, etc).
+  let result;
```

Comment:
I'd rather add an extra function for this try/catch instead of a mutable let var